packages/react/src/lib/shouldHandleOnKeys.ts

```diff
@@ -3,18 +3,26 @@ import * as keyboardKey from 'keyboard-key'
 import * as _ from 'lodash'
 import * as React from 'react'
 
+const shouldHandleModifier = (modifierValue: boolean, combinationValue?: boolean) => {
+  if (typeof combinationValue === 'undefined') {
```

Comment:
`combinationValue === undefined` should be enough here, as `combinationValue` is guaranteed to be declared (local one)